src/DateTime.cc

```diff
@@ -348,7 +348,7 @@ dafBase::DateTime::DateTime(double date, DateSystem system, Timescale scale) {
 
 
 /** Constructor.
- * \param[in] year Year number.
+ * \param[in] year Year number. Must be in the range [1961, 2261], inclusive.
```

Comment:
The lower value in the comment doesn't match the value in the code or the commit message.